esp-hal-common/src/delay.rs

```diff
@@ -95,7 +95,7 @@ mod delay {
 
     // FIXME: The ESP32-S2 and ESP32-S3 have fixed crystal frequencies of 40MHz.
     //        This will not always be the case when using the ESP32.
```

Comment:
Can you please either remove or update this comment (I had conflated crystal and clock frequencies when I made this anyways)?